cordapp-cpk2/src/main/java/net/corda/plugins/cpb2/CpbTask.java

```diff
@@ -104,4 +123,57 @@ public void checkForDuplicateCpkCordappNames() {
             }
         }
     }
+
+    public void extractTransitiveDeps() {
+        if (jarDir != null) {
+            FileCollection jars = getInputs().getFiles();
+            Set<String> jarNames = new HashSet<>();
+            for (File file : jars) {
+                jarNames.add(file.getName());
```

Comment:
Can we move this inside the next for loop, which is iterating through the same FileCollection?